vshard/storage/init.lua

```diff
@@ -2155,14 +2173,18 @@ local function bucket_send(bucket_id, destination, opts)
     if type(bucket_id) ~= 'number' or type(destination) ~= 'string' then
         error('Usage: bucket_send(bucket_id, destination)')
     end
-    M.rebalancer_transfering_buckets[bucket_id] = true
+    local ret
```

Comment:
Nit: let's make the code consistent with `bucket_recv`:

```diff
diff --git a/vshard/storage/init.lua b/vshard/storage/init.lua
index fffe6a1..899f6f0 100644
--- a/vshard/storage/init.lua
+++ b/vshard/storage/init.lua
@@ -2173,8 +2173,8 @@ local function bucket_send(bucket_id, destination, opts)
     if type(bucket_id) ~= 'number' or type(destination) ~= 'string' then
         error('Usage: bucket_send(bucket_id, destination)')
     end
-    local ret
-    local status, err = bucket_transfer_start(bucket_id)
+    local status, ret, err
+    status, err = bucket_transfer_start(bucket_id)
     if not status then
         return nil, err
     end

```

Reply:
Applied.